### PR TITLE
Adding Layer Attention 

### PR DESCRIPTION
**Pull request by atroiano**
* layer_attention function
* documentation
* Added basic test case

### CODE REVIEW

File: R/layer-attention.R [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+#' @param causal Boolean. Set to True for decoder self-attention. Adds a mask such that position i cannot attend to positions j > i. 
+#' This prevents the flow of information from the future towards the past.
+#'
+#' @family core layers
+#' @family attention layers   
+#'         
+#' @export
+layer_attention <- function(inputs,use_scale=FALSE, causal = FALSE, batch_size = NULL, dtype = NULL, 
+                                name = NULL, trainable = NULL, weights = NULL) {
+  create_layer(keras$layers$Attention, inputs, list(
```
Comment (dfalbel) [inline R/layer-attention.R]: Maybe add a check for the minimum version required for the Attention layer - if it was introduced after 1.14

Comment (atroiano) [reply R/layer-attention.R]: Ok cool! I'll make the update.

Comment (atroiano) [reply R/layer-attention.R]: Done, I added it to the test case as well

**Comment by dfalbel** [comment]
Thanks very much @atroiano ! :)

**Comment by atroiano** [comment]
No problem! Thanks for all you do!

**Pull request merged**